src/antsibull_docs/docs_parsing/ansible_doc.py

```diff
@@ -105,28 +104,26 @@ def get_collection_metadata(venv: t.Union['VenvRunner', 'FakeVenvRunner'],
     return collection_metadata
 
 
-def get_ansible_core_version(venv: t.Union['VenvRunner', 'FakeVenvRunner'],
-                             env: t.Optional[t.Dict[str, str]] = None,
-                             ) -> PypiVer:
-    try:
-        venv_python = venv.get_command('python')
-        ansible_version_cmd = venv_python(
-            '-c', 'import ansible.release; print(ansible.release.__version__)', _env=env)
-        output = ansible_version_cmd.stdout.decode('utf-8', errors='surrogateescape').strip()
+async def get_ansible_core_version(venv: t.Union['VenvRunner', 'FakeVenvRunner'],
+                                   env: t.Optional[t.Dict[str, str]] = None,
+                                   ) -> PypiVer:
+    p = await venv.async_log_run(
+        ['python', '-c', 'import ansible.release; print(ansible.release.__version__)'],
```

Comment:
I opened https://github.com/ansible-community/antsibull-core/pull/65.

Reply:
That would be a breaking change for this code, this one is actively running the first `python` in path instead of using `sys.executable` for the fake venv.

Reply:
I thought a bit more about this (and looked again at the code), and I think this change would be fine. Calling `python -c '...'` is a short-cut to avoid the slow start-up time of `ansible --version`, and if it fails (because the wrong Python is used) it will fall back to `ansible --version` anyway.

Reply:
Right. Although, now that I'm also thinking... is there a reason we don't just run `ansible.release; print(ansible.release.__version__)` directly? It seems strange to me to shell out to `python -c` from python, but maybe there's some historical reason to this.

Reply:
If the venv is not a fake venv, but a real one, you cannot simply directly import Python code from it.

Reply:
:facepalm: right....